### ISSUE TITLE
Site / Docs

### ISSUE THREAD

**Issue by svpernova09** [SimpleLance/simplelance.github.io#1]
I'd *love* to use sculpin for the site https://gist.github.com/mrkrstphr/2e79e55229b2ef7a343a is some details on how to do it with github pages. 

I'm not sure how to do the layout stuff, but I think it'd just be a matter of customizing the page types and such in the sculpin project.

**Comment by railto** [comment]
Agreed on using sculpin and GH pages (I'm moving in that direction for anything that can be static).  Layout wise shouldn't be too bad, once I get the tickets reply functionality completed I'll throw up a simple bootstrap layout that will at least get the site up and running and give us a place to start

**Comment by railto** [comment]
Beginnings of site is up, markdown files have been created for each documentation page in source/documentation/ so editing those files will fill out each page.  

Any input on site welcome as always
